frontend/src/pages/DeviceEditPage/DeviceEditPage.tsx

```diff
@@ -66,6 +66,7 @@ export const DeviceEditPage: React.FC<Props> = ({ targetDevice, targets }) => {
   if (!device) return null
 
   const thisDevice = device.id === targetDevice.uid
+  const editable = thisDevice || device.configurable
```

Comment:
are you sure that is `thisDevice || device.configurable` and not `thisDevice && device.configurable`?

Reply:
@mauricio0000 Yes. You can have full edit rights if you are on the device locally, or if the device is a CloudShift device that has the configurable flag.